.yaydoc.yml

```diff
@@ -3,26 +3,19 @@ metadata:
   projectname: "Susi Server"
   version: development
 build:
-  theme: sphinx_fossasia_theme   
-  source: docs
+  theme:
+    name: sphinx_fossasia_theme
   logo: images/susi-logo-original.svg
   subproject:
     - url: https://github.com/fossasia/susi_android
-      source: docs
     - url: https://github.com/fossasia/chat.susi.ai
-      source: docs
     - url: https://github.com/fossasia/susi_tweetbot
-      source: docs
     - url: https://github.com/fossasia/susi_fbbot
-      source: docs
     - url: https://github.com/fossasia/susi_slackbot
-      source: docs
     - url: https://github.com/fossasia/susi_desktop
       source: .
     - url: https://github.com/fossasia/susi_iOS
-      source: docs
     - url: https://github.com/fossasia/susi_telegrambot
-      source: docs
 publish:
   ghpages:
     docurl: dev.susi.ai
```

Comment:
There is too much space wasted on the left of the sidebar. Can that be avoided? Also please consider decreasing the font size of links in the sidebar(not the title) so that long names also look good.